proprietary/tokens/src/brand/ams/color.tokens.json

```diff
@@ -3,126 +3,50 @@
     "brand": {
       "color": {
         "azure": {
-          "10": { "value": "#F2F7FD" },
-          "20": { "value": "#E4EFFA" },
-          "30": { "value": "#C6DDF5" },
-          "40": { "value": "#A1CAF0" },
-          "50": { "value": "#72B5EB" },
           "60": { "value": "#009DE6" },
-          "70": { "value": "#008CCE" },
-          "80": { "value": "#007AB2" },
-          "90": { "value": "#006391" },
-          "100": { "value": "#004667" }
+          "80": { "value": "#007AB2" }
```

Comment:
We currently don't use this value, should we add it? YAGNI and all... Same for blue 20 and the other '80' values except blue and red.

Reply:
Oh wow, I forgot to remove quite a bunch of them.